Functional.Primitives.Extensions/OptionExtensions.cs

```diff
@@ -142,5 +142,21 @@ public static void Apply<TValue>(this Option<TValue> option, Action<TValue> appl
 
 		public static Task Apply<TValue>(this Task<Option<TValue>> option, Action<TValue> applyWhenSome, Action applyWhenNone)
 			=> option.Do(applyWhenSome, applyWhenNone);
+
+		[AllowAllocations]
+		public static IEnumerable<T> ToEnumerable<T>(this Option<T> option)
+			=> option.TryGetValue(out var value) ? new[] { value } : Enumerable.Empty<T>();
+
+		[AllowAllocations]
+		public static async Task<IEnumerable<T>> ToEnumerable<T>(this Task<Option<T>> option)
```

Comment:
I don't think [AllowAllocations] is required on this method.

Reply:
Correct, thanks for the catch.  Fixed.